CHANGELOG.md

```diff
@@ -0,0 +1,125 @@
+# Changelog
```

Comment:
Would this conflict with the way release changelog works through the semantic release bot? Though we should be documenting this somehow anyway.

Reply:
It doesn't seem to be adding anything in here so I think it's safe to put here